docs/version0.33.md

```diff
@@ -35,10 +35,12 @@ The following new features and notable changes since version 0.30.0 are included
 
 ### Binaries and supported environments
 
-OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, and 18.
+OpenJ9 releases 0.33.x support OpenJDK 8, 11, 17, and 18.
```

Comment:
Based on [this comment](https://github.com/eclipse-openj9/openj9-docs/pull/979/files#r949389470), this should say "release 0.33.x supports".